### ISSUE TITLE
Clippings/wx: JSON or HTML export files are blank

### ISSUE THREAD

**Issue by aecreations** [aecreations/clippings#152]
Clippings 6.0/6.0.1

Reported by several users.  Export files are empty; i.e. no content.  One affected user gave example contents of the exported files:

clippings.html
```
<!DOCTYPE html>
<html><head><meta charset="utf-8"><title>Clippings</title></head><body><h1>Clippings</h1><dl></dl></body></html>
```

clippings.json
```
{"version":"6.0","createdBy":"Clippings/wx","userClippingsRoot":[]}
```

**Comment by aecreations** [comment]
A user on the forums has reported that the following error occurs when export or backup fails:

> InvalidStateError: An attempt was made to use an object that is not, or is no longer, usable

No additional info (e.g. source file and line number) was returned by the error.

**Comment by aecreations** [comment]
Fixed by refactoring the method for exporting to JSON.  Needs to be tested by users that have experienced this issue. Will ask the user from the forums to help out.

**Comment by aecreations** [comment]
I've received news that the fix for this bug has been verified to be successful: https://groups.google.com/forum/#!category-topic/aecreations-help/EGxL1s0L3Gw

**Comment by aecreations** [comment]
Fixed in 6.0.2.1
